documentation/README.md

```diff
@@ -72,7 +72,6 @@ import axios from 'axios';
 const examplePromise = () => {
   axios.get('http://localhost:3001/example')
     .then((data) => {
-      console.warn(data);
```

Comment:
I see that someone's looking out for `console.warns`!!! I love it!

Reply:
Webpack instructions ;)